webpack.config.js

```diff
@@ -211,6 +226,10 @@ const webpackConfig = {
 		new CopyWebpackPlugin( [
 			{ from: 'node_modules/flag-icon-css/flags/4x3', to: 'images/flags' },
 		] ),
+		new MiniCssExtractPlugin( {
+			filename: '[name].css',
```

Comment:
Isn't that the default `filename`? Do we need to explicitly specify it?

Reply:
Seems like we actually need this: If I drop this line, the CSS file ends up being named `editor-notes-editor-script.css` instead of `style.css`.

Reply:
Oops, no, seems unrelated -- it's called `editor-notes-editor-script.css` even with these options in place. My bad.